File: Assignment_1/program4.c
// Accept one number and check whether it is divisible by 5 or not.

///////////////////////////////////////////////////////////////////////////////
// File Name   :  program4.c
// Description :  Accept one number and check whether it is divisible by 5 or not.
// Autor       :  Ankita Anil Patil
// Date        :  09/05/2025
///////////////////////////////////////////////////////////////////////////////


#include<stdio.h>

typedef int BOOL;
#define TRUE 1
#define FALSE 0

BOOL check(int iNo)
{
    if((iNo % 5 ) == 0)
    {
        return TRUE;

    }
    else
    {
        return FALSE;

}}

int main()
{
    int iValue = 0;
    BOOL bRet = FALSE;

    printf("Enter number ");
    scanf("%d",&iValue);

    bRet = check(iValue);

    if(bRet == TRUE )
    {
        printf("Divisible by 5\n");
    }
    else
    {
        printf("Not divisible by 5\n");
    }

    return 0;
}

File: Assignment_59/program4.java
//  Write java program to print below pattern. 
/*
      *        * 
     * *      * * 
    * * *    * * * 
   * * * *  * * * * 
  * * * * * * * * * * 
 
 */

////////////////////////////////////////////////////////////////////////////////
// File Name   :  program4.java
// Description :  Accept row from user and print above pattern
// Autor       :  Ankita Anil Patil
// Date        :  14/09/2025
///////////////////////////////////////////////////////////////////////////////


import java.util.*; 

class Pattern 
{ 

     public void PrintPattern(int rows)
    {
        int i = 0, j = 0;

        for(i = 0; i <= rows; i++)
        {
            for ( j = 1; j <= rows - i; j++) 
            {
                System.out.print(" ");
            }

            for ( j = 1; j <= i; j++) 
            {
                System.out.print("* ");
            }
           
            for ( j = 1; j < rows - i +1 ; j++) 
            {
                System.out.print("  ");
            }
            
            for ( j = 1; j <= i; j++) 
            {
                System.out.print("* ");
            }

            

            System.out.println();
        }

    }
} 
class program4
{
    public static void main(String args[]) 
    { 
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter row : ");
        int iRow = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.PrintPattern(iRow);

    } 
    
}

